contracts/MechMarketplace.sol

```diff
@@ -43,10 +43,10 @@ contract MechMarketplace is IErrorsMarketplace {
     event ImplementationUpdated(address indexed implementation);
     event MarketplaceParamsUpdated(uint256 fee, uint256 minResponseTimeout, uint256 maxResponseTimeout);
     event SetMechFactoryStatuses(address[] mechFactories, bool[] statuses);
-    event SetMechTypeEscrows(IMech.MechType[] mechTypes, address[] escrows);
+    event SetPaymentTypeBalanceTrackers(uint8[] paymentTypes, address[] balanceTrackers);
```

Comment:
`uint8` is a better solution, this way no need to modify MechMarketplace when another mech type is added.